examples/seldon_deployment/run.py

```diff
@@ -0,0 +1,209 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+import click
+from pipeline import (
+    DeploymentTriggerConfig,
+    SeldonDeployerConfig,
+    SeldonDeploymentLoaderStepConfig,
+    SklearnTrainerConfig,
+    TensorflowTrainerConfig,
+    continuous_deployment_pipeline,
+    deployment_trigger,
+    dynamic_importer,
+    importer_mnist,
+    inference_pipeline,
+    normalizer,
+    prediction_service_loader,
+    predictor,
+    seldon_model_deployer,
+    sklearn_evaluator,
+    sklearn_predict_preprocessor,
+    sklearn_trainer,
+    tf_evaluator,
+    tf_predict_preprocessor,
+    tf_trainer,
```

Comment:
While its quite awesome that you have implementations for sklearn as well as tensorflow, it adds complexity to this example. Why do you think its necessary to show off implementations for two model types?

Reply:
An important side of the Seldon Core integration is being able to deploy several model types for online prediction. Sklearn and tensorflow are both here to show that we're flexible on that end and that no code changes are required in the pipeline to accommodate this. Auto-detecting the deployer implementation from the model type is also a nice feature that I want to add to this PR.